src/protos/io/defang/v1/fabric.proto

```diff
@@ -117,6 +117,24 @@ message UploadURLRequest { string digest = 1; }
 
 message UploadURLResponse { string url = 1; }
 
+
+enum ServiceState {
+  UNKNOWN = 0;
+
+  // Build states (initial state for build)
+  BUILD_PENDING = 1; // BUILD_QUEUED -> BUILD_ACTIVATING
```

Comment:
I'm not sure why these aren't simply 1:1 the existing ones? Seems not too much gain for lot of confusion

Reply:
I was trying to map them to something platform agnostic but probably a little early to be doing that.